dvc/remote/azure.py

```diff
@@ -29,6 +29,7 @@ class RemoteAZURE(RemoteBASE):
     REQUIRES = {"azure-storage-blob": "azure.storage.blob"}
     PARAM_CHECKSUM = "etag"
     COPY_POLL_SECONDS = 5
+    LIST_OBJECT_PAGE_SIZE = 5000
```

Comment:
unused?

Reply:
It gets used in `RemoteBASE`, it's overridden here since azure uses a page size that is different than the default (1000)

Reply:
ah my bad; I didn't notice the indentation - thought it was a top-level constant rather than a class member.